stories/exercises/06_exercise.mdx

```diff
@@ -0,0 +1,48 @@
+# 06. Customize components using settings.
+
+In the last exercise, we familiarized ourselves with the USWDS theme settings. In this section, we'll take that a step further by using component specific settings.
+
+1. Visit the [banner component](https://designsystem.digital.gov/components/banner/) page.
+1. Find the "Banner settings" section.
+
+    Each USWDS component with settings will have this section. It is a snapshot of what customizations are possible without needed to write custom styles.
+
+    **Tip:** The [Component settings](https://designsystem.digital.gov/documentation/settings/#component-settings-2) section of the Settings page provides a comprehensive list of all available component settings, their default values, and what type of token or value they take!
+
+1. To get started, lets try updating the font to a serif font. In `uswds-theme.scss`, add the following line to your settings map:\
+    ```scss
+    $theme-banner-font-family: "serif",
+    ```
+1. Easy, right? Let's add a bit more color to our banner. Let's see what would happen if we brought back our accent color:
+    ```scss
+    $theme-banner-background-color: "accent-cool"
+    ```
+    The banner updated, but if you have terminal window open you might have noticed a bunch of contrast warnings pop up. USWDS does our best to ensure appropriate color contrast throughout the design system by setting good defaults. Once the default is changed, we check again where we can to ensure compliance and even try to implement a fallback color if it's not.
+
+    If the automated checks determine neither color meets contrast requirements, we'll allow you to override but give you a warning. If we inspect this warning, we learn that the link color and it's fallback do not meet color contrast minimums. Luckily, there's a setting for that!
+1. Find the setting for the banner link color and a token that we defined in exercise 5 to prevent the contrast error.
```

Comment:
**question:** How do we feel about the more "informative" guidance here? In previous exercises, we didn't include much but I could see value in the long term if we include it similar to tutorial.

Reply:
That type of guidance can be useful. We might want to prefix it with `**note:**` So that users can quickly see.

Reply:
Added this in [235b685](https://github.com/Bixal/storybook-rapid-prototyping/pull/19/commits/235b685d6f27e3033b4ce4803fc309cb42cba4f4)

@mejiaj  let me know what you think of the placement! I think this feels like the appropriate place for it.